test/evergreen.yml

```diff
@@ -121,8 +121,12 @@ functions:
           ${HAVE_DIAGNOSTIC|} \
           ${GNU_C_VERSION|} \
           ${GNU_CXX_VERSION|} \
+          ${ENABLE_AZURE|} \
+          ${ENABLE_GCP|} \
           ${ENABLE_S3|} \
           ${ENABLE_CPPSUITE|} \
```

Comment:
I know it's not from this PR but can we move `ENABLE_CPPSUITE` so it is sorted A-Z?

Reply:
done!